shared/src/main/scala/tastyquery/reader/TreeUnpickler.scala

```diff
@@ -428,200 +455,219 @@ class TreeUnpickler(protected val reader: TastyReader, nameAtRef: NameTable) {
+        val finalizer = reader.ifBefore(end)(readTerm, EmptyTree)
+        Try(expr, catchCases, finalizer)(curSpan)
+      case ASSIGN =>
+        reader.readByte()
+        reader.readEnd()
+        val lhsSpan = span
+        val lhs = readTerm
+        val rhsSpan = span
+        val rhs = readTerm
+        Assign(lhs, rhs)(lhsSpan.union(rhsSpan))
+      case BLOCK =>
+        reader.readByte()
+        val end = reader.readEnd()
+        val expr = readTerm
+        Block(readStats(end), expr)(curSpan)
+      case IF =>
+        reader.readByte()
+        reader.readEnd()
+        if (reader.nextByte == INLINE) {
+          reader.readByte()
+          new InlineIf(readTerm, readTerm, readTerm)(curSpan)
+        } else {
+          If(readTerm, readTerm, readTerm)(curSpan)
+        }
+      case LAMBDA =>
+        reader.readByte()
+        val end = reader.readEnd()
+        val method = readTerm
+        val tpt = reader.ifBefore(end)(readTypeTree, EmptyTypeTree)
+        Lambda(method, tpt)(curSpan)
+      case MATCH =>
+        reader.readByte()
+        val end = reader.readEnd()
+        if (reader.nextByte == IMPLICIT) {
+          reader.readByte()
+          new InlineMatch(EmptyTree, readCases[CaseDef](CaseDefFactory, end))(curSpan)
+        } else if (reader.nextByte == INLINE) {
+          reader.readByte()
+          new InlineMatch(readTerm, readCases[CaseDef](CaseDefFactory, end))(curSpan)
+        } else Match(readTerm, readCases[CaseDef](CaseDefFactory, end))(curSpan)
+      case BIND =>
+        val start = reader.currentAddr
+        reader.readByte()
+        val end = reader.readEnd()
+        val name = readName
+        // TODO: use type
+        val typ = readType
+        val term = readTerm
+        skipModifiers(end)
+        val symbol = ctx.getSymbol(start, RegularSymbolFactory)
+        Bind(name, term, symbol)(curSpan).definesTreeOf(symbol)
+      case ALTERNATIVE =>
+        reader.readByte()
+        val end = reader.readEnd()
+        Alternative(reader.until(end)(readTerm))(curSpan)
+      case UNAPPLY =>
+        reader.readByte()
+        val end = reader.readEnd()
+        val fun = readTerm
+        val args = reader.collectWhile(reader.nextByte == IMPLICITarg)({
+          assert(reader.readByte() == IMPLICITarg, posErrorMsg)
+          readTerm
+        })
+        // TODO: use pattern type
+        val patType = readType
+        val patterns = reader.until(end)(readTerm)
+        Unapply(fun, args, patterns)(curSpan)
+      case REPEATED =>
+        reader.readByte()
+        val end = reader.readEnd()
+        val elemType = readTypeTree
+        SeqLiteral(reader.until(end)(readTerm), elemType)(curSpan)
+      case WHILE =>
+        reader.readByte()
+        reader.readEnd()
+        While(readTerm, readTerm)(curSpan)
+      case RETURN =>
+        reader.readByte()
+        val end = reader.readEnd()
+        val from = readSymRef
+        val idSpan = span
```

Comment:
in Scala there is no labelled return so this should be an empty span

Reply:
You're right. It's fixed now.